### ISSUE TITLE
PayPal Payment on Android devices triggers the installed PayPal app

### ISSUE THREAD

**Issue by spraveenitpro** [woothemes/woocommerce#8403]
**Here is a step-by-step of the issue occurring when users on an android mobile (such as a Samsung) checkout using PayPal**

1. Users successfully add items to their cart and proceed to the checkout page 
2. Users elect Paypal as the payment option (I only accept PayPal payment) and click 'Proceed to Paypal' 
3. A window pops up saying 'Complete action with: **Internet, Chrome, PayPal**
4. If users elect to complete the action with PayPal  App (which seems to be a logical decision) they are directed to Paypal  App and receive the message **'Error Detected: Your Shopping Cart is Empty** 
5. I receive a cancelled order transaction on my end, and the custom isn't able to checkout.
6. If they elect to complete the action with Chrome, it works successfully.

With the Standard PayPal checkout is it possible to either: 

a)	Make the Paypal app receive the data and work, or 
b)	Make it so that when the ‘Completing the action with’ screen pops up, Paypal is not an option, or 
c)	Add some text near the cart page for users on an android mobile to, if prompted, complete the action using chrome?

Concerned ticket https://woothemes.zendesk.com/agent/tickets/305563
@BFTrick @ABDpromotions

**Comment by BFTrick** [comment]
I'm trying to get into contact with the PayPal Android app team.

If there's some special meta data we can add to the cart so that the app can read what's in the cart I'm fine with that. 

I really don't think we should add a warning message. It's something that the PayPal app is doing wrong. if it can't get the necessary data it shouldn't popup. It's also a very small segment of our users.

**Comment by BFTrick** [comment]
> Our team tried live Android PayPal APP with our paypal account and both went thru successfully to the PayPal Review Pages. We are not able to reproduce the issue. It looks like maybe the issue maybe device specific or user specific? The device we were using is Samsung S5.

I'm going to get some more information from the user. Specifically looking for 
* What devices & firmware are affected?
* Who is affected? The store owner or a customer?
* Is the issue still happening and if so can the store owner give us some screen shots?

**Comment by aolsonVIN** [comment]
Running into the same exact issue.
Using a Samsung GALAXY S5 here. It also seems a few customers are running into the issue, finding 'cancelled' orders. While we do not know specifically if they are using android phone or not we noticed once they run into that error they try again, until either they give up or use a different device to complete the order. 

I have investigated permissions on the server, i have made sure we have the most up to date apps on the mobile device and our website has the most recent build of Woocommerce (etc). I have checked for plugin and theme related conflicts and there seem to be none that we could find.
